src/test_client.py

```diff
@@ -0,0 +1,11 @@
+"""Test the echo server functionality."""
+# -*- coding: utf-8 -*-
+
+
+def test_client():
+    """Test client function using server in server.py."""
+    from client import client
+    assert client('short') == 'short'
+    assert client('blah') == 'blah'
+    assert client('88888888') == '88888888'
+    assert client('±¥Ä') == '±¥Ä'
```

Comment:
Great that you are testing unicode chars. Make sure to test a string where a unicode char might overlap your bufferlength e.g. `1234567Ä`